### DISCUSSION TITLE
No sampling method choice in text-to-image tab

### DISCUSSION THREAD

**Discussion by Garuspik24** [vladmandic/sdnext#3233] [Q&A]
Hello. No sampling method is available for selection on latest release of Sd.Next
![изображение_2024-06-14_164707718](https://github.com/vladmandic/automatic/assets/164914673/273105c8-31e2-4fb5-a496-9d7e778457c2)

In settings some oа samplers are selected:
![изображение_2024-06-14_164852880](https://github.com/vladmandic/automatic/assets/164914673/6dc5a2fa-b941-4a62-bdfa-193e1fbf28d6)

in Image to Image tab everything is ok
![изображение_2024-06-14_165034783](https://github.com/vladmandic/automatic/assets/164914673/9012526e-de15-4ea2-9807-acaed7eb8271)

Thanks.

**Comment by vladmandic** [comment]
if its an actual issue, then collect all required data and create an issue, not a discussion item?
